Resources/Prototypes/_Sunrise/Entities/Clothing/Head/decorative.yml

```
@@ -0,0 +1,10 @@
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadKigurumiPanda
+  name: KigurumiPanda helmet
+  description: KigurumiPanda helmet
+  components:
+  - type: Sprite
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)